### ISSUE TITLE
[Question]: Rescan subscriptions not working: 'OSError: write error'

### ISSUE THREAD

**Issue by grantterra** [tubearchivist/tubearchivist#381]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read through the [wiki](https://github.com/tubearchivist/tubearchivist/wiki) and the [readme](https://github.com/tubearchivist/tubearchivist#installing-and-updating), particularly the [common errors](https://github.com/tubearchivist/tubearchivist#common-errors) section.

### Operating System

Unraid 6.11.5

## Describe the bug
After 'Rescan subscriptions', get nothing when it finished.
![image](https://user-images.githubusercontent.com/4091143/205911186-9002a27b-cd8a-495b-9e51-63216940cf9b.png)

## Steps To Reproduce
Added new videos to a playlist that subscribed by TA, then click 'Rescan subscriptions'.

## Expected behavior
New videos appear.

### Relevant log output

```shell
[2022-12-06 07:09:33,611: WARNING/ForkPoolWorker-8] A6bbAZzXkUU: add to download queue
[2022-12-06 07:09:34,717: WARNING/ForkPoolWorker-8] ERROR: [youtube] A6bbAZzXkUU: Video unavailable. This video contains content from SME, who has blocked it in your country on copyright grounds
[2022-12-06 07:09:34,717: WARNING/ForkPoolWorker-8] A6bbAZzXkUU: failed to get info from youtube
[2022-12-06 07:09:34,717: INFO/ForkPoolWorker-8] Task update_subscribed[12fc14a9-5d74-4cb7-b988-a52e950ed955] succeeded in 136.67288361903047s: None
Tue Dec  6 07:09:49 2022 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /static/favicon/apple-touch-icon.a94db2e7a4e7.png (ip 192.168.50.3) !!!
Tue Dec  6 07:09:49 2022 - uwsgi_response_sendfile_do(): Broken pipe [core/writer.c line 655] during GET /static/favicon/apple-touch-icon.a94db2e7a4e7.png (192.168.50.3)
OSError: write error
```

### Anything else?

Just updated to v0.3.0 from v0.2.4.
Rescan subscriptions was worked on v0.2.4

Any ideas?

**Comment by grantterra** [comment]
I manually added a video to download queue via browser extension, that video is existing in the playlist I've already rescan subs before. Then I downloaded it.

Thereafter, I rescan subs again, now it finally works.

**Comment by bbilly1** [comment]
Also, in case you didn't see it in your log output, you answered your question there yourself: 

```
[2022-12-06 07:09:34,717: WARNING/ForkPoolWorker-8] ERROR: [youtube] A6bbAZzXkUU: Video unavailable. This video contains content from SME, who has blocked it in your country on copyright grounds
[2022-12-06 07:09:34,717: WARNING/ForkPoolWorker-8] A6bbAZzXkUU: failed to get info from youtube
```

Tube Archivist can't download a video if it's blocked. Sometimes adding the cookie can help.

**Comment by grantterra** [comment]
> Also, in case you didn't see it in your log output, you answered your question there yourself:
> 
> ```
> [2022-12-06 07:09:34,717: WARNING/ForkPoolWorker-8] ERROR: [youtube] A6bbAZzXkUU: Video unavailable. This video contains content from SME, who has blocked it in your country on copyright grounds
> [2022-12-06 07:09:34,717: WARNING/ForkPoolWorker-8] A6bbAZzXkUU: failed to get info from youtube
> ```
> 
> Tube Archivist can't download a video if it's blocked. Sometimes adding the cookie can help.

I just post the last few lines of the output.

I already know TA can't download a video if it's blocked, the problem is even some video is not blocked, it still doesn't appear until I manually added via browser extension.

**Comment by grantterra** [comment]
I'm sorry if this confused, I'm trying my best to describe it.

Rescan subscriptions is not working. (it seems finished, but nothing appear in the download queue, I'm pretty sure there are a dozen videos are not blocked and not downloaded yet.)

Then I manually added a video to download queue via browser extension. (the video is exists in a playlist that I've already rescan)

I click rescan again, I saw dozens of video added to the queue.

![image](https://user-images.githubusercontent.com/4091143/206135575-2b9cc209-c1e2-4e05-8ebc-3c4f47572672.png)
